src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -288,11 +293,35 @@ def _max_by_axis(self, the_list: List[List[int]]) -> List[int]:
                 maxes[index] = max(maxes[index], item)
         return maxes
 
+    def convert_segmentation_map_to_binary_masks(self, segmentation_map: "np.ndarray", num_labels: int):
+        # get all the labels in the image
+        labels = np.unique(segmentation_map)
+        # remove ignore index
+        labels = labels[labels != self.ignore_index]
+        # sanity check
+        max_label = labels.max().item()
+        if max_label > num_labels:
+            raise ValueError(f"We found label = {max_label}, but `self.num_labels` is {num_labels}.")
+        # we need to convert mask from [W,H] to [C, W, H]
+        all_labels = np.arange(num_labels)
+        # helping broadcast by making mask [1,W,H] and labels [C, 1, 1]
+        binary_masks = (
+            segmentation_map[
+                np.newaxis,
+            ]
+            == all_labels[:, np.newaxis, np.newaxis]
+        )
```

Comment:
```suggestion
        binary_masks = segmentation_map[None]  == all_labels[:, None, None]
```
We use `None` everywhere else in the lib, so let's be consistent. Plus it fits in one line.

Reply:
Neat!